verification/ci-verify.py

```
@@ -0,0 +1,1555 @@
+        for needle in must_contain_cmp:
+            if needle not in content_cmp:
+                print(f"       [content] missing required string: {needle!r}")
+                return False
+        for needle in must_not_contain_cmp:
+            if needle in content_cmp:
+                print(f"       [content] found forbidden string: {needle!r}")
+                return False
+        # When no string assertions are supplied (None or empty list), fall back
+        # to a non-empty check so an empty output file never passes vacuously.
+        if not must_contain and not must_not_contain:
+            if not content.strip():
```

Ruff (0.15.17)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)